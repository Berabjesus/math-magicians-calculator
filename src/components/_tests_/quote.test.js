import RRender from 'react-test-renderer';
import Quote from '../display/quote';

it('renders the Quote Page correctly without errors', () => {
  const tree = RRender.create(<Quote />).toJSON();

  expect(tree).toMatchSnapshot();
});
