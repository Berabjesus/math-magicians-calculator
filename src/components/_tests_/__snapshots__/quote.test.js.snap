// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Quote Page correctly without errors 1`] = `
<section
  className="pt-5"
>
  <h2
    className="text-center"
  >
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
  </h2>
</section>
`;
